src/patterns/exit-a-page-quickly/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Exit a page quickly
+description: Give users a way to quickly exit a service, website or application quickly and safely.
+section: Patterns
+theme: Help users to…
+aliases:
+backlog_issue_id: 213
+layout: layout-pane.njk
+status: Experimental
+statusMessage: This component is currently experimental. You'll need to do your own research to decide whether to use this pattern and add the <a class="govuk-link" href="/components/exit-this-page/">Exit this page</a> component to your service.
+---
+
+{% from "_example.njk" import example %}
+
+Give users a way to quickly exit a service, website or application quickly and safely.
+Use this pattern to add the [Exit this page](/components/exit-this-page/) component to your service and help users use it safely to protect their privacy.
```

Comment:
```suggestion
Use this pattern to add the [Exit this page](/components/exit-this-page/) component to your service and keep users safe by helping them to protect their privacy.
```

Reply:
This suggestion was a trial to see if I could lose the 'users use' part of the sentence.

Reply:
How about this? Feeling like we need to keep the focus of this on the user.

```suggestion
Use this pattern to add the [Exit this page](/components/exit-this-page/) component to your service, which helps users protect their privacy and keep themselves safe.
```

Reply:
Yes, that's the one